book/website/collaboration/inclusive-events/inclusive-events-schedules.md

```diff
@@ -0,0 +1,374 @@
+
+This timing can vary depending on individuals, however, both ADHD advocates and people with a lived 
+experience of cognitive impairments (such as stroke, traumatic brain injury, and brain fog) have advocated 
+for 30-40 minute session blocks for activities that require sustained attention and engagement. 
+
+Try to plan for the most important parts of your event's content to be shared within blocks of time 
+that are under 40 minutes.
+
+Because many events are planned in 1-hour or 1.5-hour timing blocks, consider choosing shorter sessions 
+or incorporating breaks within your session. 
+This can be done in addition to allocating 5-10 minutes for an introduction and session set-up. 
+This can help reduce the cognitive strain that many people with lived experience of 
+cognitive impairments are affected by. 
+
+> **Example 1: 1-hour Focus Group Event**
+    - 10 minute welcome, Code of Conduct, and Session introduction
+    - 40 minute Discussion between participants
+    - 10 minute break or 10 minute thank you and session close
+
+> **Example 2: 1-hour Workshop event**
+    - 5 minute introduction 
+    - 20 minute session
+    - 5 minute break
+    - 20 minute session
+    - 10 minute break or 10 minute thank you and session close
+
+It is best to get feedback and input from a representative with lived experience when planning your session timings. 
+This is because there may be differences in what is required for a specific community that would negatively 
+impact a different set of individuals. 
+
+For example, if your event invites specifically people with lived experience of ADHD 
+(and no other cognitive impairments) for a creative workshop. 
+You may want to consider offering longer sessions that are facilitated with self-directed breaks. 
+For example, for a 1.5-hr event you may consider a 10 minute introduction, followed by a 50 minute self-directed session, 
+a 15 minute break, and then a 15 minute conclusion or share out and session close. 
+This is because many people with a lived experience of ADHD can find that forced interruption is very disruptive and 
+prohibitive of meaningful engagement on tasks that are creative or require deep focus.
+
+(cl-inclusive-events-event-duration)=
+### Event Duration
+In addition to session length, it is also important to consider how your event will be structured to support the 
+involvement and engagement of people with lived experience. 
+
+If you are organizing a multi-day conference, consider picking either a single day or a repeating timeslot 
+that features any content specifically important for people with lived experience. 
+For example, Day 1 of a 3-Day event might schedule the majority of sessions focused on public engagement and lived experience, 
+followed by Day 2 which could focus on technical workshops and other content. 
+Alternatively, sessions from 10 a.m. - 1 p.m. on all 3 days could focus on lived experience content. 
```

Comment:
For this I'm not sure about how good that would work in practice. At least if that's the only content for PPI members it means they have to travel/get accommodations, carers etc. every day. But I guess that goes back to "ask the people with lived experience what works for them" instead of assuming :)

Reply:
Added in a sentence as I think it always bears repeating.